integration/benchmark/proto/generate.go

```diff
@@ -0,0 +1,3 @@
+//go:generate protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative macro_benchmark.proto
```

Comment:
i'm not familiar with the proto generation setup.  should this be part of the make file instead?